jest.config.js

```diff
@@ -1,14 +1,15 @@
 module.exports = {
   rootDir: 'spec',
   transformIgnorePatterns: [
-    'node_modules/(?!(@18f/identity-address-search)/)',
+    'node_modules/(?!(@18f/identity-address-search|@18f/identity-i18n)/)',
     '\\.pnp\\.[^\\/]+$',
   ],
   moduleNameMapper: {
     '@18f/identity-address-search':
       '<rootDir>/../node_modules/@18f/identity-address-search/dist/index.js',
+    '@18f/identity-i18n': '<rootDir>/../node_modules/@18f/identity-i18n/dist/index.js',
```

Comment:
This is probably fine for this usage right now. Ideally we'd upgrade Jest in this repo so these can be properly imported.